source/cmake/CompilerOptions.cmake

```diff
@@ -21,6 +21,8 @@ if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
             sign-conversion                 # Happens a lot in these samples, would complicate them too much to handle manually
             sign-compare                    # Happens a lot in these samples, would complicate them too much to handle manually
             shorten-64-to-32                # Narrowing conversions: Too strict and noisy for this code base
+            padded                          # The type and order of elements caused the compiler to add padding to the end of a struct; specifically: RollPitchYaw::convention and Representations::rotationVector
+            conversion                      # Implicit conversion loses integer precision (signed to unsigned): Expected to happen a lot in these samples, would complicate them too much to handle manually; specifically: pointCloudDownsampled(i, j).setRgb(redDownsampled(i, j), greenDownsampled(i, j), blueDownsampled(i, j));
```

Comment:
I think the information about which code triggered these warnings belong in the commit message rather than in the code. Comments in a commit message are about the state of the code as that commit was made, whereas comments in the code are expected to keep being up to date with the current state of the code. 

So here in the `.cmake` file, I'd just have
```cmake
padded # The type and order of elements caused the compiler to add padding to the end of a struct
conversion # Implicit conversion loses integer precision (signed to unsigned). Expected to happen a lot in these samples, would complicate them too much to handle manually

# (...)

4820 # The type and order of elements caused the compiler to add padding to the end of a struct
```

And then move the rest of the info to the commit message,

Reply:
done